src/structures/Chat.js

```diff
@@ -112,6 +118,21 @@ class Chat extends Base {
         return this.client.unarchiveChat(this.id._serialized);
     }
 
+    /**
+     * mutes this chat
+     * @param {Number} timestamp
```

Comment:
Can you describe what this time stamp refers to?

Reply:
Timestamp is the exparation time

Reply:
Oh, can it really be any arbitrary date not in those intervals?

Also, maybe this should be a JS date and do the time stamp conversion internally for convenience.